web/styles/message_view_header.css

```diff
@@ -119,4 +119,50 @@
             vertical-align: middle;
         }
     }
+
+    .narrow_description_extended {
+        overflow-wrap: break-word;
+        hyphens: auto;
+        overflow: hidden;
+        white-space: normal;
+        background-color: var(--color-background-navbar);
+        border-radius: 5px;
+        animation: expand-extended-description 250ms ease-in both;
+
+        &.leaving {
```

Comment:
`leaving` is too generic a class name; `leaving-extended-narrow-description` seems OK.